src/hermit.rs

```diff
@@ -115,4 +133,33 @@ mod tests {
         assert_eq!(*first_op,
                    Op::GitInit(PathBuf::from("/home/geoff/.hermit-config/shells/new-one")));
     }
+
+    #[test]
+    fn can_inhabit_existing_shell() {
+        let config = MockConfig {
+            root_path: PathBuf::from(".config"),
+            allowed_shell_names: vec!["default".to_owned(), 
+                "new-one".to_owned()],
+            current_shell: "default".to_owned(),
+        };
+        let mut hermit = hermit(&config);
+        let mut file_ops = FileOperations::rooted_at("/home/houngj"); 
+        hermit.inhabit_shell(&mut file_ops, "new-one");
+        let shell = hermit.current_shell().unwrap();
```

Comment:
I think rather than "unsafely" calling unwrap (we don't _know_ that the result is `Ok`) we should do something that fails with a better error message.  A good option would be to unwrap with `expect` and give it a nice error message.